### ISSUE TITLE
Staff Bros. in Custom 

### ISSUE THREAD

**Issue by Tmi489** [smogon/pokemon-showdown#6819]
In the tour help  (github.com/smogon/pokemon-showdown/blob/master/config/CUSTOM-RULES.md) It says "ban made-up things other than CAP (such as Magikarp's Revenge, or Staff Bros moves)"
Right now, enabling +Custom (or using custom game) allows the former, but does not allow for staffbros items. From the description it seems like staff bros moves/abilities should be available for custom game use. Fix the description (which is just magikarp's revenge then) or add the stuff.
There may be a few things that need to be addressed (like Next . Level Strats + L9999)

**Comment by Zarel** [comment]
Yeah, it probably should just say "Magikarp's Revenge". Staff Bros is currently implemented as a mod, which can't be enabled just with rules.
